### DISCUSSION TITLE
Cannot Import my Bank data, maybe I misunderstand something?

### DISCUSSION THREAD

**Discussion by GeorgFoc** [firefly-iii/firefly-iii#6580] [Firefly III]
Hi there,
i really like the Idea an optic of firefly. It was setup quite easy. 
Now I wanted to start importing Bank Transactions. I created an account. And than I thought I can start importing data. 

So I got an CSV file. Here is an example:
```IBAN;BIC;Betrag;Buchungstext;Buchungstag;Begünstigter/Absender - Bankleitzahl;Begünstigter/Absender - Kontonummer;Begünstigter/Absender - Name;Kategorie;Saldo;Saldo - Währung;Unterkategorie;Budget;Verwendungszweckzeile 1;Verwendungszweckzeile 2;Verwendungszweckzeile 3;Verwendungszweckzeile 4;Verwendungszweckzeile 5;Verwendungszweckzeile 6;Verwendungszweckzeile 7;Verwendungszweckzeile 8;Verwendungszweckzeile 9;Verwendungszweckzeile 10;Verwendungszweckzeile 11;Verwendungszweckzeile 12;Verwendungszweckzeile 13;Verwendungszweckzeile 14;Wertstellungstag;Abweichender Auftraggeber
DExxxx00000000xxxxx;BxxxxxxxS;-129;FOLGELASTSCHRIFT;1.10.2020;ByyyyyyyyyX;DEyyyyyy0000000yyyyyy0;Allg.Deutscher Automobil-Club ADAC e.V.;Versicherung;1479,83;EUR;Versicherung;Versicherungen;ADAC E.V. NAME; 01.10.20-01.10.21;;;;;;;;;;;;;;1.10.2020;ADAC, Hansastr. 19, 80686 München
DExxxx00000000xxxxx;BxxxxxxxS;-
```
Than I tried to import it. the data-importer works. But I never see something imported. In the log files, both data importer and server I don't see an Error. A lot of information, but no Error. 

Do I have to create all accounts, categories etc before I import?
 
Is there somewhere a tutorial how to use Firefly III by example? 

I read in the Docs:

> Although the form allows you to free-type a destination account as well, this may fail and the transaction can't be saved.

So do I have to create a destination Account for all my transactions? I read that I can combine many destinations into one account, but than I would have to do some auto mapping? 

So if someone can give me a hint, for the right direction, It would be nice. Or a good Tutorial / Example etc.

**Comment by JC5** [comment]
I'm curious to hear what your process is. After uploading the file, did you get through all the screens until the "success" screen?

**Reply by GeorgFoc** [reply]
Yes. The Import went through with success. When I am back at home tonight (German time) I can, if wanted, upload some logs. Maybe someone notice something, I oversee.

**Reply by JC5** [reply]
So when it is finished, it gives you direct links to the transactions it imported, yes?

**Reply by GeorgFoc** [reply]
No there is not such a link :thinking: 

Maybe I have to recheck the logs line by line.
